modules/oe_theme_content_person/oe_theme_content_person.module

```diff
@@ -0,0 +1,101 @@
+<?php
+
+/**
+ * @file
+ * Module file used for theming Person content type.
+ */
+
+declare(strict_types = 1);
+
+use Drupal\Core\Entity\Display\EntityViewDisplayInterface;
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\oe_theme\ValueObject\ImageValueObject;
+use Drupal\oe_content_person\PersonNodeWrapper;
+
+/**
+ * Prepares variables for in-page navigation.
+ *
+ * Default template: oe-theme-helper-in-page-navigation.html.twig.
+ *
+ * @param array $variables
+ *   An array of variables.
+ */
+function oe_theme_content_person_preprocess_oe_theme_helper_in_page_navigation__node__oe_person(array &$variables): void {
+  _oe_theme_helper_prepare_image_from_media_field($variables, $variables['entity'], 'oe_person_photo');
+
+  if (empty($variables['image'])) {
+    // Set default image.
+    $image_path = base_path() . drupal_get_path('theme', 'oe_theme') . '/images/user_icon.svg';
+    $variables['image'] = ImageValueObject::fromArray(['src' => $image_path]);
+  }
+}
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function oe_theme_content_person_preprocess_field__node__oe_person_organisation__oe_person(&$variables) {
+  foreach ($variables['items'] as $key => $item) {
+    $node = $variables['items'][$key]['content']['#node'];
+    // Since Entity reference revision module doesn't provide formatter to show
+    // label of the referenced entity, we will prepare it here.
+    $variables['items'][$key]['content'] = [
+      '#markup' => $node->label(),
```

Comment:
To be clear: I decided not to use the patch, given the time constraints that we used to have when this was under work, I haven't actually suggested the approach used here.

Reply:
After discussing it with @yenyasinn we concluded that it's worth trying out the patch at comment **[#31](https://www.drupal.org/project/entity_reference_revisions/issues/2937835#comment-12442457)** (not the last one), as this will allow to remove this function completely and, possibly, also the extra field on the person job.